webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayNode.scala

```diff
@@ -806,31 +804,51 @@ object DisplayNode extends Loggable {
       // special: add name -> value to be displayed as a table
       Full(
         Seq(
-          ("Node ID", escape(sm.node.main.id.value)),
-          ("Hostname", escape(sm.node.main.hostname)),
-          ("Policy Server ID", escape(sm.node.main.policyServerId.value)),
-          ("Operating System Detailed name", escape(sm.node.main.osDetails.fullName)),
-          ("Operating System Type", escape(sm.node.main.osDetails.os.kernelName)),
-          ("Operating System Name", escape(S.?("os.name." + sm.node.main.osDetails.os.name))),
-          ("Operating System Version", escape(sm.node.main.osDetails.version.value)),
-          ("Operating System Service Pack", escape(sm.node.main.osDetails.servicePack.getOrElse("None"))),
-          ("Operating System Architecture Description", escape(sm.node.archDescription.getOrElse("None"))),
-          ("Kernel Version", escape(sm.node.main.osDetails.kernelVersion.value)),
-          ("Total Physical Memory (RAM)", escape(sm.node.ram.map(_.toStringMo).getOrElse("-"))),
-          ("Manufacturer", escape(sm.machine.flatMap(x => x.manufacturer).map(x => x.name).getOrElse("-"))),
+          ("Node ID", StringEscapeUtils.escapeEcmaScript(sm.node.main.id.value)),
```

Comment:
is it actually JS here? and not in other tabs?

Reply:
nope :)